pkg/vcs/linux_sanitizers.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package vcs
+
+import (
+	"regexp"
+
+	"github.com/google/syzkaller/pkg/debugtracer"
+	"github.com/google/syzkaller/pkg/kconfig"
+)
+
+type linuxSanitizer struct {
+	name      string
+	titleRe   *regexp.Regexp
+	disable   func(cf *kconfig.ConfigFile)
+	dependsOn []*linuxSanitizer
+}
+
+var linuxSanitizers = []*linuxSanitizer{
+	warnings,
+	{
+		name:    "ubsan",
```

Comment:
Can some of this be moved to pkg/report? It already has report Type and knows about different sanitizer messages.

There are also LOCKDEP, LIST_DEBUG, ODEBUG, FAULT_INJECTION, etc that can be  removed based on report.